array_methods_lab.js

```diff
@@ -0,0 +1,185 @@
+// console.log('-----QUESTION 1------')
+// /*iven an array of numbers, write a function that
+// returns a new array where every element is incremented by 10.*/
+//
+// let numArr = [1,2,3,4,5]
+// let addTen = numArr.map((crrElem) => {
+//   return crrElem +  10
+// })
+// console.log(addTen)
+
```

Comment:
remember we are creating functions.  (es6)
const addTen = (arr) => {
let addTen = numArr.map((crrElem) => {
	return crrElem +  10
 })

so we have to return something. You've returned within the map but haven't returned outside.

here's how we can fix that:
const addTen = (arr) => {
return numArr.map((crrElem) => {
	return crrElem +  10
 })